src/administration/Administration.Service/BusinessLogic/CompanyDataBusinessLogic.cs

```diff
@@ -267,9 +266,9 @@ public async Task CreateCompanyCertificate(CompanyCertificateCreationData data,
             throw new ControllerArgumentException("ValidTill date should be greater than current date");
         }
 
-        if (data.Issuer != null && !Company.IsMatch(data.Issuer!))
+        if (!string.IsNullOrEmpty(data.Issuer) && !ValidationExpressionsValidator.IsValidCompanyName(data.Issuer))
```

Comment:
```suggestion
        if (data.Issuer != null && !data.Issuer.IsValidCompanyName())
```
(see suggested change of IsValidCompanyName into a static extension-method)